Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://tensorflow.pypi.thoth-station.ninja/index/manylinux2010/AVX2/simple"
+verify_ssl = true
+name = "tensorflow-pypi-thoth-station-ninja"
+
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi-org"
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
+
+[packages.tensorflow]
+index = "tensorflow-pypi-thoth-station-ninja"
+version = "==2.0.0"
```

Comment:
Do we need to have it for 2.0.0 specifically?

Reply:
to be as update as possible with tensorflow?

Reply:
Yes, also to show that we are capable of resolving large stacks (?)

Reply:
What about this one? Should we keep it?

Reply:
sorry, I will update it :D